hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/PreHandleContext.java

```diff
@@ -220,6 +221,19 @@ PreHandleContext requireKeyIfReceiverSigRequired(
     @NonNull
     PreHandleContext requireSignatureForHollowAccount(@NonNull final Account hollowAccount);
 
+    /**
+     * Adds the given hollow account to the required signing set. If the account has already been added, then
+     * the call is a no-op. The account must not be null. During signature verification, the app will verify that the
+     * transaction was signed by an ECDSA(secp256k1) key corresponding to the given account's alias. Since the account
+     * is being created, we just
+     *
+     * @param hollowAccountAlias the EVM address alias
+     * @return {@code this} object
+     * @throws IllegalArgumentException if the account is not a hollow account
+     */
+    @NonNull
+    PreHandleContext requireSignatureForHollowAccountCreation(@NonNull final Bytes hollowAccountAlias);
```

Comment:
You added this method just to avoid changing the `AccountID` assertion done in `requireSignatureForHollowAccount()`?

Reply:
No, seems reasonable to me!